src/core/Radio/index.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import { FormControlLabel } from "@material-ui/core";
+import { action } from "@storybook/addon-actions";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import RadioButton from "./index";
+
+const actions = {
+  onChange: action("onChange"),
+};
+
+const storyColumn = {
+  alignItems: "flex-start",
+  display: "flex",
+  flexDirection: "column",
+};
+
+storiesOf("Radio Button", module).add("unchecked", () => (
+  <div style={storyColumn as React.CSSProperties}>
+    <p>
+      <strong>Accessibility</strong>: All radio buttons should have a label.
+      This can be done with a visible form label or with the aria-label
+      attribute. Note aria labels should be meaningful based on your content.{" "}
+      <br /> Good: &ldquo;XYZ Gene&ldquo;
+      <br /> Bad: &ldquo;Unchecked&ldquo; <br />{" "}
+      https://material-ui.com/components/radio-buttons/#accessibility
+    </p>
+    <FormControlLabel
+      control={<RadioButton onChange={actions.onChange} stage="unchecked" />}
+      label="Unchecked"
+    />
+    <FormControlLabel
+      control={
+        <RadioButton disabled onChange={actions.onChange} stage="unchecked" />
+      }
+      label="Unchecked & disabled"
+    />
+    With aria labels:
+    <RadioButton
+      inputProps={{ "aria-label": "radio button unchecked" }}
+      onChange={actions.onChange}
+      stage="unchecked"
+    />
+    <RadioButton
+      disabled
+      inputProps={{ "aria-label": "radio button unchecked and disabled" }}
+      onChange={actions.onChange}
+      stage="unchecked"
+    />
+  </div>
+));
+
+storiesOf("Radio Button", module).add("checked", () => (
+  <div style={storyColumn as React.CSSProperties}>
+    <FormControlLabel
```

Comment:
@mdjmoore can you help investigate a little on what's causing the text and radio button to be misaligned? Thanks so much!!

<img width="209" alt="Screen Shot 2021-09-07 at 11 32 27 AM" src="https://user-images.githubusercontent.com/6309723/132394523-061a4081-cb79-446c-9347-e0178fec5567.png">

Reply:
found the problem. Mui was changing the viewbox from `0 0 16 16` to `0 0 24 24` when loading the svg, meaning it sat in the upper left of a larger box. Hard coding this in at the component level is the first part of the solution. I also had to define a set height and width for the icon-root, as seen in the `IconButton` css. 

The same problem can be seen on the checkbox:
![Screen Shot 2021-09-07 at 3 18 34 PM](https://user-images.githubusercontent.com/35696944/132399167-17ed6bea-e7ea-4b63-a4b8-536a9316ee61.png)
I will make another ticket for this. We need to make sure that addressing this bug doesn't push a breaking change to projects that have implemented the checkbox component. 

16 x 16 is not a size that exists in our default theme yet, but is used for both icon inputs. I propose adding it to the iconSizes definition in the default theme:

```  iconSizes: {
    input: { height: 16, width: 16 },
    l: { height: 22, width: 22 },
    s: { height: 14, width: 14 },
    xl: { height: 32, width: 32 },
    xs: { height: 10, width: 10 },
  },

Reply:
I think adding input icon as a size is an interesting idea.